server/app/views/questiontypes/DropdownQuestionRenderer.java

```diff
@@ -38,22 +38,24 @@ protected DivTag renderInputTag(
     SingleSelectQuestion singleSelectQuestion = applicantQuestion.createSingleSelectQuestion();
 
     SelectWithLabel select =
-        new SelectWithLabel()
-            .addReferenceClass("cf-dropdown-question")
-            .setFieldName(singleSelectQuestion.getSelectionPath().toString())
-            .setAriaRequired(!isOptional)
-            .setPlaceholderText(messages.at(MessageKey.DROPDOWN_PLACEHOLDER.getKeyName()))
-            .setOptions(
-                singleSelectQuestion.getOptions().stream()
-                    .sorted(Comparator.comparing(LocalizedQuestionOption::order))
-                    .map(
-                        option ->
-                            SelectWithLabel.OptionValue.builder()
-                                .setLabel(option.optionText())
-                                .setValue(String.valueOf(option.id()))
-                                .build())
-                    .collect(ImmutableList.toImmutableList()))
-            .setAriaDescribedByIds(ariaDescribedByIds);
+        (SelectWithLabel)
```

Comment:
(non-blocking) Rather than casting, I'd suggest overriding the method in `SelectWithLabel`, like this other method, to make full use of inheritance:    
```
@Override
  public SelectWithLabel setAriaDescribedByIds(ImmutableList<String> ariaDescribedByIds) {
    super.setAriaDescribedByIds(ariaDescribedByIds);
    return this;
  }
```
  
  You won't have to do this, though, if you remove the `maybeFocusOnInput` method, like I suggest in my other comment.

Reply:
done